src/Common/src/Interop/Windows/mincore/Interop.MoveFileEx.cs

```diff
@@ -13,6 +14,8 @@ internal partial class mincore
 
         internal static bool MoveFile(string src, string dst)
         {
+            src = PathInternal.AddExtendedPathPrefixForLongPaths(src);
+            dst = PathInternal.AddExtendedPathPrefixForLongPaths(dst);
```

Comment:
I didn't want to hunt down all of the usages to normalize p/Invoke naming in this PR- I'll do that as a separate PR to keep the functional changes isolated.